services/admin/test/utils.ts

```diff
@@ -26,18 +22,16 @@ export function buildOfficialCandidatesWriteInAdjudicationOptionGroup(
 }
 
 /**
- * Builds an auth instance for tests
+ * Builds a mock auth instance
  */
-export function buildTestAuth(): {
-  auth: DippedSmartCardAuthApi;
-  card: MockCard;
-} {
-  const card = new MockCard();
-  const auth = new DippedSmartCardAuthWithMemoryCard({
-    card,
-    config: {
-      allowElectionManagersToAccessUnconfiguredMachines: false,
-    },
-  });
-  return { auth, card };
+export function buildMockAuth(): DippedSmartCardAuthApi {
+  return {
+    getAuthStatus: jest.fn(),
+    checkPin: jest.fn(),
+    logOut: jest.fn(),
+    programCard: jest.fn(),
+    unprogramCard: jest.fn(),
+    setElectionDefinition: jest.fn(),
+    clearElectionDefinition: jest.fn(),
+  };
```

Comment:
Realized that I could just set up a fully mocked auth instance instead of a real auth instance with a mocked card reader. No need to test the internals of the auth lib here (and in every other Vx machine) when it's going to test itself. And might as well make it easier for us to mock auth states, especially as we start checking auth on the backend. We still have integration tests to test the system holistically.

What initially got me to revisit this was a warning in the test output:

```
 worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks.
```

This was due to the polling within the auth lib. I was initially ideating ways to turn off the polling in these tests when I realized that I didn't need the internals of the auth lib at all here.

Reply:
Nice, that sounds good. I almost suggested this change in your first PR but I didn't have a compelling reason